ui/src/api/index.ts

```diff
@@ -42,6 +42,19 @@ rdf.resourceFactory.mixins.push(ReferenceAttributeMixin)
 // Tells Hydra to use current browser location as base URI for relative URIs
 Hydra.baseUri = window.location.href
 
+// Inject the access token in all requests if present
+Hydra.defaultHeaders = () => {
```

Comment:
@tpluscode Is this function called for every request? If yes, then it looks good to me

Reply:
Yes, before every `loadResource()` and `operation#invoke()`